docs/source/smartdevice.rst

```diff
@@ -45,6 +45,30 @@ Simple example script showing some functionality:
     if __name__ == "__main__":
         asyncio.run(main())
 
+If you are connecting to a newer KASA or TAPO device you can get the device via discovery or connect directly with DeviceConfig:
```

Comment:
`DeviceConfig` should be a reference (to make it clickable).

Reply:
Done